src/ripple/protocol/STObject.h

```diff
@@ -401,7 +401,7 @@ class STObject
         if (!cf)
             throw std::runtime_error ("Wrong field type");
 
-        cf->setValue (value);
+        cf->setValue (std::move (value));
```

Comment:
Currently `value` is a by-value parameter and this is the last use of `value`, so yes, this is safe, *unless* the client explicitly specifies `V` to be a reference type.  I did a search on the clients and no client currently calls this function with an explicit template parameter for `V` (everybody deduces it).

Clients are also not taking advantage of move semantics when passing `value` in, but that is arguably a separate jira ticket.